frontend/language/src/nb.json

```diff
@@ -1316,6 +1337,7 @@
   "ux_editor.component_title.RadioButtons": "Radioknapper",
   "ux_editor.component_title.RepeatingGroup": "Repeterende gruppe",
   "ux_editor.component_title.Summary": "Oppsummering",
+  "ux_editor.component_title.Summary2": "Oppsummering2",
```

Comment:
Burde vi vurdere å visuelt kalle komponenten noe mer beskrivende? Husker vi såvidt diskuterte det på den ekstraordinære refinnementen vi hadde, men husker ikke hva vi landet på 🤔 

Ser for øvrig for meg at den heter noe sånt som:
`Avansert oppsummering`
`Fullstending oppsummering`
`Fleksibel oppsummering`

Reply:
Godt poeng, men husker et alternativ vi diskuterte var å la Summary2 overta for Summary når vi skrur på featuren. Da vil brukerne fortsatt kunne bruke gamle Summary-komponenter som de allerede har lagt til i skjema, mens nye Summary-komponenter som legges inn, vil være av type Summary2. Skal vi la navnet stå fram til det nærmer seg lansering?

Reply:
Jaa, det kan vi! Vil du sjekke om vi har et punkt i en taskliste for det? 🫣

Reply:
Jepp, det et punkt som heter "Review: design and text review of labels and UX." 😎